### PR TITLE
[Bugfix-2.0.x] Updated german translation

### PR DESCRIPTION
**Pull request by Ludy87**

### CODE REVIEW

File: Marlin/src/lcd/language/language_de.h [hunk 1]
```diff
@@ -39,55 +39,147 @@
...
+#define MSG_RETRACT                         _UxGT("Rückzug")
+#define MSG_MOVE_AXIS                       _UxGT("Achse Bewegen")
 #define MSG_BED_LEVELING                    _UxGT("Bett Nivellierung")
 #define MSG_LEVEL_BED                       _UxGT("Bett nivellieren")
 #define MSG_LEVEL_CORNERS                   _UxGT("Ecken nivellieren")
 #define MSG_NEXT_CORNER                     _UxGT("Nächste Ecke")
 #define MSG_EDITING_STOPPED                 _UxGT("Netzbearb. angeh.")
 #define MSG_USER_MENU                       _UxGT("Benutzer Menü")
-
+#define MSG_UBL_DOING_G29                   _UxGT("G29 auführen")
```
Comment (Zwaubel) [inline Marlin/src/lcd/language/language_de.h]: This, most probably should be "G29 ausführen"

File: Marlin/src/lcd/language/language_de.h [hunk 2]
```diff
@@ -39,55 +39,147 @@
...
+#define MSG_UBL_MESH_EDIT                   _UxGT("Netz bearbeiten")
+#define MSG_UBL_EDIT_CUSTOM_MESH            _UxGT("Eigenes Netz bearb.")
+#define MSG_UBL_FINE_TUNE_MESH              _UxGT("Feineinstellung...")
+#define MSG_UBL_DONE_EDITING_MESH           _UxGT("Bearbeitung beendet")
+#define MSG_UBL_BUILD_CUSTOM_MESH           _UxGT("Eigenes Netz erst.")
+#define MSG_UBL_BUILD_MESH_MENU             _UxGT("Netz erstellen")
+#define MSG_UBL_BUILD_PLA_MESH              _UxGT("Netz erstellen PLA")
+#define MSG_UBL_BUILD_ABS_MESH              _UxGT("Netz erstellen ABS")
+#define MSG_UBL_BUILD_COLD_MESH             _UxGT("Netz erstellen kalt")
+#define MSG_UBL_MESH_HEIGHT_ADJUST          _UxGT("Netz Höhe einst.")
```
Comment (Zwaubel) [inline Marlin/src/lcd/language/language_de.h]: I think, "Netzhöhe einst." sounds a little better

File: Marlin/src/lcd/language/language_de.h [hunk 3]
```diff
@@ -39,55 +39,147 @@
...
+#define MSG_UBL_MESH_HEIGHT_AMOUNT          _UxGT("Höhe")
+#define MSG_UBL_VALIDATE_MESH_MENU          _UxGT("Netz validieren")
+#define MSG_UBL_VALIDATE_PLA_MESH           _UxGT("Netz validieren PLA")
+#define MSG_UBL_VALIDATE_ABS_MESH           _UxGT("Netz validieren ABS")
+#define MSG_UBL_VALIDATE_CUSTOM_MESH        _UxGT("Eig. Netz validieren")
+#define MSG_UBL_CONTINUE_MESH               _UxGT("Netzerst. forts.")
+#define MSG_UBL_MESH_LEVELING               _UxGT("Netz Nivellierung")
+#define MSG_UBL_3POINT_MESH_LEVELING        _UxGT("3-Punkt Nivellierung")
```
Comment (Zwaubel) [inline Marlin/src/lcd/language/language_de.h]: Perhaps "3-Punkt-Nivellierung" ?

File: Marlin/src/lcd/language/language_de.h [hunk 4]
```diff
@@ -39,55 +39,147 @@
...
+#define MSG_UBL_GRID_MESH_LEVELING          _UxGT("Gitternetz Nivell.")
```
Comment (Zwaubel) [inline Marlin/src/lcd/language/language_de.h]: Same here "Gitternetz-Nivell."

File: Marlin/src/lcd/language/language_de.h [hunk 5]
```diff
@@ -233,117 +336,36 @@
...
-#define MSG_INFO_PRINTER_MENU               _UxGT("Drucker Info")
+#define MSG_INFO_PRINTER_MENU               _UxGT("Drucker Infos")
 #define MSG_3POINT_LEVELING                 _UxGT("3-Punkt Nivellierung")
 #define MSG_LINEAR_LEVELING                 _UxGT("Lineare Nivellierung")
 #define MSG_BILINEAR_LEVELING               _UxGT("Bilineare Nivell.")
 #define MSG_UBL_LEVELING                    _UxGT("Unified Bed Leveling")
 #define MSG_MESH_LEVELING                   _UxGT("Netz Nivellierung")
 #define MSG_INFO_STATS_MENU                 _UxGT("Drucker Statistik")
-#define MSG_INFO_BOARD_MENU                 _UxGT("Board Info")
+#define MSG_INFO_BOARD_MENU                 _UxGT("Board Infos")
```
Comment (Zwaubel) [inline Marlin/src/lcd/language/language_de.h]: I'm not sure, but I do believe that "Info" is correct already...

File: Marlin/src/lcd/language/language_de.h [hunk 6]
```diff
@@ -39,55 +39,147 @@
...
+#define MSG_UBL_MANUAL_MESH                 _UxGT("Netz manuell erst.")
+#define MSG_UBL_BC_INSERT                   _UxGT("Unterlegen & messen")
+#define MSG_UBL_BC_INSERT2                  _UxGT("Messen")
+#define MSG_UBL_BC_REMOVE                   _UxGT("Entfernen & messen")
+#define MSG_UBL_MOVING_TO_NEXT              _UxGT("Nächster Punkt...")
+#define MSG_UBL_ACTIVATE_MESH               _UxGT("UBL aktivieren")
+#define MSG_UBL_DEACTIVATE_MESH             _UxGT("UBL deaktivieren")
+#define MSG_UBL_SET_BED_TEMP                _UxGT("Bett Temp.")
+#define MSG_UBL_CUSTOM_BED_TEMP             MSG_UBL_SET_BED_TEMP
+#define MSG_UBL_SET_HOTEND_TEMP             _UxGT("Hotend Temp.")
```
Comment (Zwaubel) [inline Marlin/src/lcd/language/language_de.h:110]: To unify the things, I'd say that we either accept "Hotend" or use "Düse" as a translation, respectively "Hotend-Temp." or "Düsentemp.".

File: Marlin/src/lcd/language/language_de.h [hunk 7]
```diff
@@ -39,55 +39,147 @@
...
+#define MSG_IDEX_Z_OFFSET                   _UxGT("2. Z Düse")
+#define MSG_IDEX_SAVE_OFFSETS               _UxGT("Versätze speichern")
+#define MSG_UBL_MANUAL_MESH                 _UxGT("Netz manuell erst.")
+#define MSG_UBL_BC_INSERT                   _UxGT("Unterlegen & messen")
+#define MSG_UBL_BC_INSERT2                  _UxGT("Messen")
+#define MSG_UBL_BC_REMOVE                   _UxGT("Entfernen & messen")
+#define MSG_UBL_MOVING_TO_NEXT              _UxGT("Nächster Punkt...")
+#define MSG_UBL_ACTIVATE_MESH               _UxGT("UBL aktivieren")
+#define MSG_UBL_DEACTIVATE_MESH             _UxGT("UBL deaktivieren")
+#define MSG_UBL_SET_BED_TEMP                _UxGT("Bett Temp.")
```
Comment (Zwaubel) [inline Marlin/src/lcd/language/language_de.h]: How about "Betttemp."

File: Marlin/src/lcd/language/language_de.h [hunk 8]
```diff
@@ -233,117 +336,36 @@
 #define MSG_DELTA_SETTINGS                  _UxGT("Delta Einst. anzeig.")
 #define MSG_DELTA_AUTO_CALIBRATE            _UxGT("Autom. Kalibrierung")
 #define MSG_DELTA_HEIGHT_CALIBRATE          _UxGT("Delta Höhe setzen")
+#define MSG_DELTA_Z_OFFSET_CALIBRATE        _UxGT("Sondenversatz Z")
 #define MSG_DELTA_DIAG_ROD                  _UxGT("Diag Rod")
 #define MSG_DELTA_HEIGHT                    _UxGT("Höhe")
 #define MSG_DELTA_RADIUS                    _UxGT("Radius")
 #define MSG_INFO_MENU                       _UxGT("Über den Drucker")
-#define MSG_INFO_PRINTER_MENU               _UxGT("Drucker Info")
+#define MSG_INFO_PRINTER_MENU               _UxGT("Drucker Infos")
```
Comment (Zwaubel) [inline Marlin/src/lcd/language/language_de.h]: Same here: I think it is "Info"

**Review by Zwaubel — COMMENTED** [review]
great work, @Ludy87.
I reviewed your translations and added a few comments. What do you think?

File: Marlin/src/lcd/language/language_de.h (hunk 6 above)
Comment (Ludy87) [reply Marlin/src/lcd/language/language_de.h:110]: Hotend and nozzle is not a same

Comment (Zwaubel) [reply Marlin/src/lcd/language/language_de.h:110]: You're right. I forgot about that...

File: Marlin/src/lcd/language/language_de.h (hunk 1 above)
Comment (Ludy87) [reply Marlin/src/lcd/language/language_de.h]: Oh No, sorry.

File: Marlin/src/lcd/language/language_de.h (hunk 8 above)
Comment (Ludy87) [reply Marlin/src/lcd/language/language_de.h]: Plural from Info is Infos or Informationen

Comment (kAdonis) [reply Marlin/src/lcd/language/language_de.h]: I like "Info" more

File: Marlin/src/lcd/language/language_de.h [hunk 9]
```diff
@@ -151,65 +243,74 @@
...
+#define MSG_POWER_LOSS_RECOVERY             _UxGT("Wiederh. n. Stroma.")
+#define MSG_CARD_MENU                       _UxGT("Druck v. SD-Karte")
 #define MSG_NO_CARD                         _UxGT("Keine SD-Karte")
 #define MSG_DWELL                           _UxGT("Warten...")
-#define MSG_USERWAIT                        _UxGT("Warte auf Nutzer")
+#define MSG_USERWAIT                        _UxGT("Klick zum fortsetzen")
 #define MSG_PRINT_PAUSED                    _UxGT("Druck pausiert")
+#define MSG_PRINTING                        _UxGT("Druckt...")
 #define MSG_PRINT_ABORTED                   _UxGT("Druck abgebrochen")
 #define MSG_NO_MOVE                         _UxGT("Motoren eingeschaltet")
```
Comment (thinkyhead) [inline Marlin/src/lcd/language/language_de.h]:
```
12345678901234567890
benutzerdef. Vorheizen
Motoren eingeschaltet
Bett heizen Erfolglos
HEIZEN FEHLGESCHLAGEN
REDUND. TEMPERATURABWEICHUNG
```

**Comment by thinkyhead** [comment]
Why do you suppose this now shows 150 files changed?

**Comment by thinkyhead** [comment]
Rebased and squashed

**Pull request merged**